### PR TITLE
Bug 4499/fixing measurement periods

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE PR

Jira Ticket: [MAT-4499](https://jira.cms.gov/browse/MAT-4499)
(Optional) Related Tickets:

### Summary
Fixed the bug, where measurement periods are taking extra characters 
 
### All Submissions
* [x] This PR has the JIRA linked.
* [ ] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [x] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/NewMeasure/CreateNewMeasureDialog.tsx [hunk 1]
```diff
@@ -369,22 +369,22 @@ const CreateNewMeasureDialog = ({ open, onClose }) => {
...
-                      "measurementPeriodStart",
-                      true
-                    )}
-                    error={
-                      formik.touched.measurementPeriodStart &&
-                      Boolean(formik.errors.measurementPeriodStart)
-                    }
-                    {...formik.getFieldProps("measurementPeriodStart")}
-                    data-testid="measurement-period-start"
-                  />
```
Comment (RohitKandimalla) [inline src/components/NewMeasure/CreateNewMeasureDialog.tsx:386]: Do we have another PR to fix the same issue in madie-measure while a measure is being edited?

File: src/components/NewMeasure/CreateNewMeasureDialog.tsx [hunk 2]
```diff
@@ -369,22 +369,22 @@ const CreateNewMeasureDialog = ({ open, onClose }) => {
...
+                      {...formikFieldProps}
+                      {...params}
+                      required
+                      data-testid="measurement-period-start"
+                      helperText={formikErrorHandler(
+                        "measurementPeriodStart",
+                        true
+                      )}
+                    />
+                  );
```
Comment (RohitKandimalla) [inline src/components/NewMeasure/CreateNewMeasureDialog.tsx:386]: Apparently, we don't have any test suite for the CreateNewMeasure component, Can we create a test suite and add these specific test cases, I believe we need another Tech debt story to write all test scenarios.

File: src/components/NewMeasure/CreateNewMeasureDialog.tsx (hunk 1 above)
Comment (sb-prateekkeerthi) [reply src/components/NewMeasure/CreateNewMeasureDialog.tsx:386]: Yup we will have a PR for that I am working on it

File: src/components/NewMeasure/CreateNewMeasureDialog.tsx (hunk 2 above)
Comment (sb-prateekkeerthi) [reply src/components/NewMeasure/CreateNewMeasureDialog.tsx:386]: I have added the test cases

File: src/components/NewMeasure/CreateNewMeasureDialog.test.tsx [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+      const submitButton = await screen.getByTestId(
+        "create-new-measure-save-button"
+      );
+      await waitFor(() => expect(submitButton).toBeDisabled(), {
+        timeout: 3000,
+      });
+    });
+  }, 20000);
+
+  test("checking if error text is displayed when measurement periods are not provided", async () => {
```
Comment (RohitKandimalla) [inline src/components/NewMeasure/CreateNewMeasureDialog.test.tsx]: Looks like there are 2 test cases with the same description.

**Review by RohitKandimalla — CHANGES_REQUESTED** [review]

File: src/components/NewMeasure/CreateNewMeasureDialog.test.tsx (hunk 3 above)
Comment (sb-prateekkeerthi) [reply src/components/NewMeasure/CreateNewMeasureDialog.test.tsx]: thank you, I just changed it

**Review by RohitKandimalla — APPROVED** [review]
